zstash/extract.py

```diff
@@ -182,18 +182,23 @@ def extract(keep_files=True):
             cur.execute(u"select value from config where arg=?", (attr,))
             value = cur.fetchone()[0]
             setattr(config, attr, value)
-    # FIXME: No overload variant of "int" matches argument type "None" mypy(error)
-    config.maxsize = int(config.maxsize)  # type: ignore
-    # FIXME: Incompatible types in assignment (expression has type "bool", variable has type "None")mypy(error)
-    config.keep = bool(int(config.keep))  # type: ignore
+    if config.maxsize:
```

Comment:
I noticed that this chunk of code isn't in `extract.py` like in `create.py`.
```
    optional.add_argument(
        "--maxsize",
        type=float,
        help="maximum size of tar archives (in GB, default 256)",
        default=256,
    )
```
I'm assuming `config.maxsize` will always be `None` since it is never assigned through the command line argument.

If so, do we need to check for `config.maxsize` still? We might get away with deleting this check altogether.